src/TesApi.Web/Startup.cs

```diff
@@ -138,9 +138,9 @@ public void ConfigureServices(IServiceCollection services)
                     .AddHostedService<DeleteCompletedBatchJobsHostedService>()
                     .AddHostedService<DeleteOrphanedBatchJobsHostedService>()
                     .AddHostedService<DeleteOrphanedAutoPoolsHostedService>();
-                    //.AddHostedService<RefreshVMSizesAndPricesHostedService>()
+                //.AddHostedService<RefreshVMSizesAndPricesHostedService>()
```

Comment:
Nit: This looks weird. @MattMcL4475 git blame tells me this comment line is yours, can it be deleted?

Reply:
It's a placeholder RN because it appears that the PriceApi is still making calls out for each new task even though it "warmed itself up" when TES started.